stream-chat-android-ui-components/api/stream-chat-android-ui-components.api

```diff
@@ -813,14 +813,10 @@ public final class io/getstream/chat/android/ui/message/list/adapter/MessageList
 
 public class io/getstream/chat/android/ui/message/list/adapter/MessageListItemViewHolderFactory {
 	public fun <init> ()V
-	protected final fun createAttachmentsViewHolder (Landroid/view/ViewGroup;)Lio/getstream/chat/android/ui/message/list/adapter/BaseMessageItemViewHolder;
```

Comment:
Changes to public API here should be in the CHANGELOG in some form - though I expect very few people are already customizing things with the new UI, this is still breaking

Reply:
Fixed